.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+target
```

Comment:
I just realize I didn't escape asterisks in my first comment. Please take a look to it.